src/app/fdctl/config/default.toml

```diff
@@ -1,585 +1,663 @@
+    # instance as a fallback to local ledger data.  The
+    # `GOOGLE_APPLICATION_CREDENTIALS` environment variable must be set
+    # to access BigTable.
+    #
+    # This option is passed to the Solana Labs client with the
+    # `--enable-rpc-bigtable-ledger-storage` argument.
     bigtable_storage = false
 
-    # If nonempty, enable an accounts index indexed by the specified field. The
-    # account field must be one of "program-id", "spl-token-owner", or
-    # "spl-token-mint". These options are passed to the Solana Labs client with
-    # the `--account-index` argument.
+    # If nonempty, enable an accounts index indexed by the specified
+    # field.  The account field must be one of "program-id",
+    # "spl-token-owner", or "spl-token-mint".  These options are passed
+    # to the Solana Labs client with the `--account-index` argument.
     account_indexes = []
 
-    # If account indexes are enabled, exclude these keys from the index. These
-    # options are passed to the Solana Labs client with the
+    # If account indexes are enabled, exclude these keys from the index.
+    # These options are passed to the Solana Labs client with the
     # `--account-index-exclude-key` argument.
     account_index_exclude_keys = []
 
-    # Whether to use compression when storing snapshots. This option is passed
-    # to the Solana Labs client with the `--snapshot-archive-format` argument.
+    # Whether to use compression when storing snapshots.  This option is
+    # passed to the Solana Labs client with the
+    # `--snapshot-archive-format` argument.
     snapshot_archive_format = "zstd"
 
-    # Refuse to start if saved tower state is not found. This option is passed
-    # to the Solana Labs client with the `--require-tower` argument.
+    # Refuse to start if saved tower state is not found.  This option is
+    # passed to the Solana Labs client with the `--require-tower`
+    # argument.
     require_tower = false
 
 [gossip]
-    # Routable DNS name or IP address and port number to use to rendezvous with
-    # the gossip cluster. These entrypoints are passed to the Solana Labs client
-    # with the `--entrypoint` argument.
+    # Routable DNS name or IP address and port number to use to
+    # rendezvous with the gossip cluster.  These entrypoints are passed
+    # to the Solana Labs client with the `--entrypoint` argument.
     entrypoints = []
 
-    # If true, on startup the validator checks that it can connect to all of the
-    # entrypoints on the given port. If not true, the validator will exit. This
-    # option is passed to the Solana Labs client inverted with the
-    # `--no-port-check` argument.
+    # If true, on startup the validator checks that it can connect to
+    # all of the entrypoints on the given port.  If not true, the
+    # validator will exit.  This option is passed to the Solana Labs
+    # client inverted with the `--no-port-check` argument.
     port_check = true
```

Comment:
If true, checks that at least one of the provided entrypoints can connect to this validator on all necessary ports.

The code is confusingly named and not documented, but `do_verify_reachable_ports` sends a ping request to a remote host and listens locally to make sure it can receive the ping.